src/templates/workCategoryTags.js

```diff
@@ -8,7 +8,7 @@ import TagSelector from '../components/TagSelector'
 import SectionItemList from '../components/SectionItemList'
 import {
   capitalize,
-  getCategoryTags,
+  getAllTags,
```

Comment:
This file is not used anymore, can you delete it?

Reply:
Done - thanks